docusaurus.config.js

```diff
@@ -242,13 +242,13 @@ module.exports = {
         },
         gtag: {
           // You can also use your "G-" Measurement ID here.
-          // trackingID: "x", // For local dev purposes
+          trackingID: "x", // For local dev purposes
```

Comment:
-! You need to change these back to their production values.  Either you can not commit the docusaurus config when making changes, or you can swap these values back when you finish a PR, but they can't "sit" on the local dev values they have to sit on the production values.

Reply:
Fixed.